ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,898 @@
+
+.. _beginner-cloud:
+
+Tuturial: |ccloud| CLI
+=======================
+
+Overview
+--------
+
+This tuturial will show you how to use the `Confluent Cloud CLI
```

Comment:
Is this intro okay, or we need a little more meat?

Reply:
FYC: add the bulleted list of steps that is covered 

```
* Create a new environment and specify it as the default
 * Create a new Kafka cluster and specify it as the default		
 * Create a user key/secret pair and specify it as the default		
 * Produce and consume with Confluent Cloud CLI		
 * Create a service account key/secret pair		
 * Run a Java producer: before and after ACLs		
 * Run a Java producer: showcase a Prefix ACL		
 * Run Connect and kafka-connect-datagen connector with permissions		
 * Run a Java consumer: showcase a Wildcard ACL		
 * Delete the API key, service account, Kafka topics, Kafka cluster, environment, and the log files
```